assets/scripts/enable-opt-out.js

```diff
@@ -5,6 +5,6 @@
   $('.core-standards-opt-out-link').on('click', function (e) {
     e.preventDefault();
     gaOptout();
```

Comment:
Not caused by this PR: The toggle link does not allow to undo the operation (re-enable the tracking after opting out). This should be fixed in the future.